README.md

```diff
@@ -229,6 +229,7 @@ data:
   ip-10-0-135-96.us-east-2.compute.internal: |
     OVN_KUBE_LOG_LEVEL=5
     OVN_LOG_LEVEL=dbg
+    OVS_LOG_LEVEL=dbg
```

Comment:
should we call this `OVS_SYS_LOG_LEVEL`? That makes it more explicit, since you could set console and file logging too.

Reply:
Sure, explicit is always better